src/function/pragma/pragma_queries.cpp

```diff
@@ -58,10 +58,27 @@ string PragmaFunctionsQuery(ClientContext &context, const FunctionParameters &pa
 
 string PragmaShow(ClientContext &context, const FunctionParameters &parameters) {
 	// PRAGMA table_info but with some aliases
-	return StringUtil::Format(
-	    "SELECT name AS \"column_name\", type as \"column_type\", CASE WHEN \"notnull\" THEN 'NO' ELSE 'YES' "
-	    "END AS \"null\", NULL AS \"key\", dflt_value AS \"default\", NULL AS \"extra\" FROM pragma_table_info('%s');",
-	    parameters.values[0].ToString());
+
+	// Remove leading and trailing quotes from string
+	auto table_name = parameters.values[0].ToString();
+	StringUtil::RemoveFirstAndLastIfMatching(table_name, '"');
```

Comment:
Can we use `QualifiedName::Parse` here instead? That is what `pragma_table_info` uses internally - and also allows us to support tables in different schemas, e.g.:

```sql
create schema s1;
create table s1.tbl(i int);
create table tbl(j int);
describe s1.tbl;
┌─────────────┬─────────────┬─────────┬───────┬─────────┬───────┐
│ column_name │ column_type │  null   │  key  │ default │ extra │
│   varchar   │   varchar   │ varchar │ int32 │ varchar │ int32 │
├─────────────┼─────────────┼─────────┼───────┼─────────┼───────┤
│ i           │ INTEGER     │ YES     │  NULL │ NULL    │  NULL │
└─────────────┴─────────────┴─────────┴───────┴─────────┴───────┘
describe tbl;
┌─────────────┬─────────────┬─────────┬───────┬─────────┬───────┐
│ column_name │ column_type │  null   │  key  │ default │ extra │
│   varchar   │   varchar   │ varchar │ int32 │ varchar │ int32 │
├─────────────┼─────────────┼─────────┼───────┼─────────┼───────┤
│ j           │ INTEGER     │ YES     │  NULL │ NULL    │  NULL │
└─────────────┴─────────────┴─────────┴───────┴─────────┴───────┘
```

Could we also add that as a test case?